.github/workflows/ci_tests.yaml

```diff
@@ -109,7 +108,9 @@ jobs:
       # Run the tests
       - name: Test with pytest
         shell: bash -l {0}
-        run: make test PYTEST_EXTRA="-r P"
+        run: |
+          python pygmt/tests/test_image_debug.py
```

Comment:
Ok, seems to be working again. Could you please remove this debug script, and change the title to just say 'Re-enable Windows builds'?

Reply:
Oops, spoke too soon. Windows build failing on test_grdimage_over_dateline!

Reply:
How do you know it fails on test_grdimage_over_dateline?

Reply:
The last build (before you force pushed) ran into a segfault I think. Now it just segfaults on test_image.

Reply:
> Now it just segfaults on test_image.

That's why I added the test_image_debug.py script. It ran fine. Perhaps we should also let test_image "runfirst"?

Reply:
Still doesn't work. We need to fix this upstream @seisman, all these workarounds are getting out of hand. I can help to finalize #485 so we can test GMT master, but tracking down the problem in the C code will be out of my area of expertise.